packages/fxa-settings/src/lib/channels/firefox.ts

```diff
@@ -209,6 +236,7 @@ export class Firefox extends EventTarget {
         detail,
       })
     );
+    this.saveEventForTests(command, data);
```

Comment:
Could this explain some of the flaky webchannel tests?

Reply:
It's definitely possible. 😅